lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -148,6 +148,10 @@ class HttpApiEvents {
       };
       resource.DependsOn = this.provider.naming.getHttpApiLogGroupLogicalId();
     }
+    this.cfTemplate.Outputs.HttpApi = {
```

Comment:
Sorry for confusion, after a second thought, I think it will be better to keep the same convention as for `Url` and output it as `HttpApiId` instead of `HttpApi`.

Reply:
that makes sense, I'll change it